src/ui/public/agg_types/controls/ip_ranges.html

```diff
@@ -53,23 +75,34 @@
     <div class="hintbox" ng-show="aggForm.rangeLength.$invalid">
       <p>
         <i class="fa fa-danger text-danger"></i>
-        <strong>Required:</strong> You must specify at least one IP range.
+        <strong
+          i18n-id="common.ui.aggTypes.ipRanges.fromTo.requiredIpRangeLabel"
+          i18n-default-message="Required:"
+        ></strong>
+        <span
+          i18n-id="common.ui.aggTypes.ipRanges.fromTo.requiredIpRangeDescription"
+          i18n-default-message=" You must specify at least one IP range."
```

Comment:
We don't need a leading space here, and should remove it. HTML will automatically have a space between those two elements, since there is a whitespace character (new line) between them.